client/bridge_client.go

```diff
@@ -172,6 +172,34 @@ func (ec *Client) BridgeGetSentChainTxsLimit(ctx context.Context) (uint64, error
 	return result, err
 }
 
+// BridgeRegisteredOperators can register an operator.
+func (ec *Client) BridgeRegisterOperator(ctx context.Context, bridge, operator common.Address) (common.Hash, error) {
+	var result common.Hash
+	err := ec.c.CallContext(ctx, &result, "subbridge_registerOperator", bridge, operator)
+	return result, err
+}
+
+// BridgeGetRegisteredOperators can return the list of operators.
+func (ec *Client) BridgeGetRegisteredOperators(ctx context.Context, bridge common.Address) (uint8, error) {
```

Comment:
What about BridgeGetOperators??
Are there UnRegisteredOperators??

Reply:
Yes. there are unregisted operators. :) For example, sub-bridge always have parent/child operator.
But all operator are not registered in bridge contract.

Reply:
thanks for explanation :)